public/js/app/templates/bookmarkletTemplate.handlebars

```diff
@@ -101,6 +105,22 @@
   <div class="row">
     <div class="col-md-12">
       <h1>Share on FreeFeed</h1>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-12 select2-container">
```

Comment:
You should use `col-xs-12` instead of `cols-md-12`, then you don't need `width: 100%` in `select2-container`. Actually, it makes sense to change it to `cols-xs-12` in all rows in this template.